gradle/dependencies.gradle

```diff
@@ -1,6 +1,6 @@
 def versions = [
     minAndroidPlugin      : '3.4.2',
-    androidPlugin         : '3.6.2',
+    androidPlugin         : '4.1.1',
```

Comment:
Can we keep AGP `3.6.2` for now as IntelliJ cannot open projects using `4.1.1` and IntelliJ deals with the multiplatform part better than Android Studio. 
I don't think there's any drawback keeping the old version of the AGP. It's mainly a `compile-only` dependency so it'll be substituted at runtime for the actual version.

Reply:
Sounds good, I'll revert it back. The main reason why I upgraded it was because the Gradle plugin was transitively pulling it in anyway, but that's likely an issue on my part. I'll also probably need to make it a `compileOnly` dependency for the plugin itself.

Reply:
Sounds good, I'll revert it back. The main reason why I upgraded it was because the new Gradle plugin was transitively pulling it in anyway, but that's likely an issue on my part. I'll also probably need to make it a compileOnly dependency for the plugin itself.